generic_config_updater/change_applier.py

```diff
@@ -160,15 +162,24 @@ def apply(self, change):
             log_error("Failed to apply Json change")
         return ret
 
-
     def remove_backend_tables_from_config(self, data):
         for key in self.backend_tables:
             data.pop(key, None)
 
-
     def _get_running_config(self):
         (_, fname) = tempfile.mkstemp(suffix="_changeApplier")
-        os.system("sonic-cfggen -d --print-data > {}".format(fname))
+
+        command = "sonic-cfggen -d --print-data"
+
+        if self.namespace is not None and self.namespace != multi_asic.DEFAULT_NAMESPACE:
+            command += " -n {}".format(self.namespace)
+        command += " > {}".format(fname)
+
+        ret_code = os.system(command)
```

Comment:
>os.system [](http://example.com/codeflow?start=19&length=9)

`os.system` is inscure. @maipbui to review.

Reply:
@maipbui Should this insecure function be caught by semgrep or CodeQL checkers?

Reply:
previously it can prevent this rule, but it seems the current default ruleset no longer includes this rule. I raised a PR adopt a different ruleset that cover more rules https://github.com/sonic-net/sonic-utilities/pull/3259

Reply:
We have many usage for os.system, are we going to replace them all?

Reply:
After discussing offline, we will make the changes for os.system in future pr since the change to subprocess will break other UTs.